### PR TITLE
Add notes for 2022-06-21

### PR DESCRIPTION
**Pull request by dschuff**

### CODE REVIEW

**Comment by dschuff** [comment]
I'm not completely sure the examples all came through.

File: main/2022/CG-06-21.md [hunk 1]
```diff
@@ -27,14 +27,75 @@ Installation is required, see the calendar invite.
...
+
+AR: I could draw an example if that helps. One thing to note that we already had similar decision with numbers in the past, possibly Dan Gohman who pointed out that we would parse 0$x as two tokens, we fixed that we would pass it as one token. Similarly the thing we would need to fix for strings are: 
+0”a”
+“A”0
+“A””b”
+Do we want to fix that to be consistent, but it would technically be backwards incompatible. Tested this in the Spec format, but didn’t bring up anything.
+
+PP: What was the use case for the first one?
+
+AR: it was really just meant for clarity. We allow lots of characters in keywords and tokens, even special characters (e.g. 0a0.52*xx4)
```
Comment (rossberg) [inline main/2022/CG-06-21.md:70]:
```suggestion
AR: it was really just meant for clarity. We allow lots of characters in keywords and tokens, even special characters, but e.g. 0a0.52*xx4 would parse as two tokens 0 and a0.52*xx4 before, which was rather surprising.
```

File: main/2022/CG-06-21.md [hunk 2]
```diff
@@ -27,14 +27,75 @@ Installation is required, see the calendar invite.
...
+AR: No, nothing special there. The only thing we allow without spaces are brackets/parens. Other than we always have to separate with white space, an oversight was that we don’t do that with strings
+
+PP: Right now, we would have two buckets, instead of a single string we would have a token?
+
+AR: Currently “a”0 would be 2 tokens, but if we make this change it would be rejected.
+
+PP: Unless someone was passing a string and a number, and they forgot to put the space in were expecting the parser to put the space in which would no longer work.
+
+AR: in the webassembly syntax, you can’t really pass things to parse as input. You could e.g. have multiple strings  in data, e.g
+(data_ “a””b”)
```
Comment (rossberg) [inline main/2022/CG-06-21.md:83]:
```suggestion
(data _ “a””b”)
```

**Review by rossberg — COMMENTED** [review]
A couple of refinements.

**Review by rossberg — APPROVED** [review]

**Comment by dschuff** [comment]
LGTM, thanks!

**Pull request merged**